test/integ/axpy/src/axpy.cpp

```diff
@@ -124,7 +124,7 @@ TEMPLATE_LIST_TEST_CASE("axpy", "[axpy]", TestAccs)
     std::random_device rd{};
     auto const seed = rd();
     std::default_random_engine eng{seed};
-    std::uniform_real_distribution<Val> dist(0.0, 1.0);
+    std::uniform_real_distribution<Val> dist(static_cast<Val>(0.0), static_cast<Val>(1.0));
```

Comment:
I would prefer a shorter version, but as you prefer.
```suggestion
    std::uniform_real_distribution<Val> dist(Val{0}, Val{1});
```